JournalMonitor/JournalMonitor.cs

```diff
@@ -1514,9 +1514,66 @@ public static List<Event> ParseJournalEntry(string line)
+                                int? scanstage = JsonParsing.getOptionalInt(data, "ScanStage");
+                                string ship = JsonParsing.getString(data, "Ship");
+                                if (ship != null)
+                                {
+                                    if (ship.Contains("fighter"))
+                                    {
+                                        string shipLocalised = JsonParsing.getString(data, "Ship_Localised");
+                                        if (shipLocalised != null)
+                                        {
+                                            ship = shipLocalised + " Fighter";
+                                        }
+                                    }
+                                    else
+                                    {
+                                        Ship shipDef = ShipDefinitions.FromEDModel(ship);
+                                        if (shipDef != null)
+                                        {
+                                            ship = shipDef.model;
+                                        }
+                                    }
+                                }
+                                string name = JsonParsing.getString(data, "PilotName_Localised");
+                                string rank = JsonParsing.getString(data, "PilotRank");
+                                if (rank != null)
+                                {
+                                    CombatRating rating = CombatRating.FromEDName(rank);
+                                    rank = rating.localizedName;
+                                }
+                                decimal? shieldHealth = JsonParsing.getOptionalDecimal(data, "ShieldHealth");
+                                decimal? hullHealth = JsonParsing.getOptionalDecimal(data, "HullHealth");
+                                string faction = JsonParsing.getString(data, "Faction");
+                                string legalStatus = JsonParsing.getString(data, "LegalStatus");
+                                if (legalStatus != null)
+                                {
+                                    switch (legalStatus)
+                                    {
+                                        case "Clean":
+                                            legalStatus = Constants.LEGALSTATUS_CLEAN;
+                                            break;
+                                        case "Wanted":
+                                            legalStatus = Constants.LEGALSTATUS_WANTED;
+                                            break;
+                                        case "Warrant":
+                                            legalStatus = Constants.LEGALSTATUS_WARRANT;
+                                            break;
+                                    }
```

Comment:
Missing legal status `Lawless`? (see example script for ShipTargetted event)

Reply:
Added LegalStatus.cs nad associated .resx files.